Trim keywords of keyword metadata

guardrex opened the pull request:
It's not super-critical that we do this, since parsers are supposed to trim the tokens of these strings, but why send the bytes? :smile:

[@]mairaw seemed to be making these consistent over in dotnet/docs, so I wondered if you'd like to do it globally over here. If not, please close this.

dnfclas commented:
@guardrex,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

dnfclas commented:
@guardrex,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

guardrex commented:
@scottaddie *DAH!* ... here I go breaking things again! :grinning:

I may have surfaced a CI bug :beetle:.

> An exception occurred during the build process. Please check the raw log and retry building your content again. If the issue persists, please open a ticket in http://SiteHelp and include the URL of publishing report.

scottaddie commented:
@guardrex Gauntlet does complain about the spaces, so it's a great change. Thank you for taking the time to do this!

guardrex commented:
> taking the time 

```csharp
var colonSplit = line.Split(":");
var keywordList = colonSplit[1].Split(',').Select(keyword => keyword.Trim()).ToArray();
outputLines.Add("keywords: " + string.Join(",", keywordList));
```

... no prob ... it took about 2 seconds (well ... 2 sec + 5 minutes). *I got mad scripts!* :grinning:
